src/components/MyArea/MyArea.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { withRouter, NavLink, RouteComponentProps } from 'react-router-dom';
+import styles from './MyArea.module.scss';
+import { MAP_URL } from './MyArea.constants';
+import { AppRoutes } from 'App.constants';
+import { ReactComponent as Logo } from 'assets/images/logo-amsterdam.svg';
+import { ReactComponent as CloseIcon } from 'assets/icons/Close.svg';
+import Heading from 'components/Heading/Heading';
+
+export const MyAreaHeader = () => {
+  return (
+    <div className={styles.Header}>
+      <Logo />
+      <h1>Mijn Buurt</h1>
+
+      <NavLink to={AppRoutes.ROOT} className={styles.CloseBtn}>
+        Sluit kaart
+        <CloseIcon />
+      </NavLink>
+    </div>
+  );
+};
+
+export const MyAreaMap = () => {
+  return (
+    <iframe
+      id="mapIframe"
+      title="Map of my area"
+      src={MAP_URL}
+      className={styles.Map}
+    />
+  );
+};
+
+const MyArea: React.SFC<RouteComponentProps> = ({ history }) => {
```

Comment:
Let's use the `function FunctionName() { ... }` notation instead of the `const FunctionName = () => { ... }`

Reply:
I Agree